css/main.css

```diff
@@ -78,6 +62,10 @@ ul li {
 p {
   padding-bottom: 5px;
 }
+.w-600 {
```

Comment:
Bootstrap has class `font-weight-bold` that could help you avoid creating an additional class.